contractsV5/bridges/bancor/mock/MockBancorNetwork.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity 0.5.11;
+
+import "../../../IERC20.sol";
+import "./IBancorNetwork.sol";
+import "../../../UtilsV5.sol";
+
+contract MockBancorNetwork is IBancorNetwork, Utils {
+
+    IERC20 public bancorETH;
+    IERC20 public bancorBNT;
+
+    uint public rateEthToBnt;
+    uint public rateBntToETh;
+
+    constructor(address _bancorETH, address _bancorBNT) public {
+        bancorETH = IERC20(_bancorETH);
+        bancorBNT = IERC20(_bancorBNT);
+    }
+
+    function() external payable { }
+
+    function setExchangeRate(uint _rateEthToBnt, uint _rateBntToEth) public {
+        rateEthToBnt = _rateEthToBnt;
+        rateBntToETh = _rateBntToEth;
+    }
+
+    function getReturnByPath(IERC20[] calldata _path, uint256 _amount) external view returns (uint256, uint256) {
+        if (_path.length != 3) { return (0, 0); }
```

Comment:
please revert on amount 0

Reply:
Added revert when amount = 0

Reply:
please revert.
we shouldn't get path with amount 0